pkg/api/deploymentapi/search.go

```diff
@@ -20,6 +20,8 @@ package deploymentapi
 import (
 	"context"
 	"errors"
+	"github.com/elastic/cloud-sdk-go/pkg/util/ec"
+	"strings"
```

Comment:
ultra nitpick: this breaks the previous grouping of imports

Reply:
Nice catch, too used to just letting the IDE handle the imports